src/models/dummy/model.cpp

```diff
@@ -1,6 +1,6 @@
 #include "model.h"
 
-#include <cmath>
+#include <numbers>
```

Comment:
Thank you for this solution.

While we're at it, could you do the same thing to `src/models/kokoro/model.cpp` and `src/util.cpp` please? Then try removing `#define _USE_MATH_DEFINES` from `src/util.h`